app/js/mms/controllers/tree.controller.js

```diff
@@ -3,13 +3,18 @@
 /* Controllers */
 
 angular.module('mmsApp')
-.controller('TreeCtrl', ['$anchorScroll' , '$q', '$filter', '$location', '$uibModal', '$scope', '$rootScope', '$state','$timeout', 'growl', 
+.controller('TreeCtrl', ['$anchorScroll' , '$q', '$filter', '$location', '$uibModal', '$scope', '$rootScope', '$state','$timeout', 'growl',
                           'UxService', 'ElementService', 'UtilsService', 'ViewService', 'ProjectService', 'MmsAppUtils', 'documentOb', 'viewOb',
                           'orgOb', 'projectOb', 'refOb', 'refObs', 'groupObs', 'docMeta',
-function($anchorScroll, $q, $filter, $location, $uibModal, $scope, $rootScope, $state, $timeout, growl, 
+function($anchorScroll, $q, $filter, $location, $uibModal, $scope, $rootScope, $state, $timeout, growl,
     UxService, ElementService, UtilsService, ViewService, ProjectService, MmsAppUtils, documentOb, viewOb,
     orgOb, projectOb, refOb, refObs, groupObs, docMeta) {
 
+    $scope.filterInputPlaceholder = 'Filter groups/docs';
+    if ($state.current.name === 'project.ref.document.view') {
```

Comment:
also test on full doc mode

Reply:
@chhaymenghong doesn't work when you go directly to a doc in full doc mode. once you update i'll merge. thanks

Reply:
@shakeh done.